src/fields.ts

```diff
@@ -207,18 +302,26 @@ export interface ImageField extends Record<string, unknown> {
 }
 
 // Links
-
+/**
+ * Link Types
+ */
 export enum LinkType {
 	Any = "Any",
 	Document = "Document",
 	Media = "Media",
 	Web = "Web",
 }
 
+/**
+ * For link fields that haven't been filled
+ */
```

Comment:
This is a type where we can use `@typeParam` to explain what the `Type` generic is.

```
@typeParam Type - The type of link.
```